src/shared/components/DashboardEditor/DashboardConfigEditor.tsx

```diff
@@ -3,6 +3,7 @@ import { Input, Form, Tooltip, Icon, Transfer, Button, Collapse } from 'antd';
 import { WrappedFormUtils } from 'antd/lib/form/Form';
 import { ResourceList } from '@bbp/nexus-sdk';
 import { FormComponentProps } from 'antd/es/form';
+import TextArea from 'antd/lib/input/TextArea';
```

Comment:
can we do import { TextArea } from 'antd' ?

Reply:
nope :(